plugins/network-elements/nuage-vsp/src/com/cloud/network/resource/NuageVspResource.java

```diff
@@ -424,9 +449,10 @@ private Answer executeRequest(ApplyAclRuleVspCommand cmd) {
     private Answer executeRequest(ShutDownVpcVspCommand cmd) {
         try {
             isNuageVspElementLoaded();
-            _nuageVspElementClient.shutDownVpc(cmd.getDomainUuid(), cmd.getVpcUuid());
+            _nuageVspElementClient.shutdownVpc(cmd.getDomainUuid(), cmd.getVpcUuid(), cmd.getDomainTemplateName());
             return new ShutDownVpcVspAnswer(cmd, true, "Shutdown VPC " + cmd.getVpcUuid());
         } catch (Exception e) {
```

Comment:
Our client is using internal exceptions, which are not available in ACS, and which translated to <code>Exception</code> before we return from the client. Therefore we need to catch on <code>Exception</code>